### PR TITLE
Complete task 4

### PR DESCRIPTION
**Pull request by Kenpachi2**
please kindly review @promise-j @ChidiPatrick

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+            fight or flight response, and activating the parasympathetic nervous
+            system, which is responsible for rest and relaxation. Yoga is an
+            ancient practice that combines physical, mental, and spiritual
+            elements to help individuals achieve a state of inner peace and
+            balance. This website explores the use of yoga as a form of
+            meditation and provides additional resources for those interested in
+            learning more.
+        </p>
+        <ul>
+            <li>
```
Comment (promise-J) [inline html/index.html:43]: From this line, you didn't nest your elements well. How can you put element li inside and outside. Please look at your code again

**Review by promise-J — CHANGES_REQUESTED** [review]
Please check your code. You are definitely doing something wrong

File: html/index.html (hunk 1 above)
Comment (Kenpachi2) [reply html/index.html:43]: Done. Can i merge now?

**Review by promise-J — APPROVED** [review]
You are cleared to move on to the next task

**Pull request merged**